README.md

```diff
@@ -1,16 +1,58 @@
-This directory contains the programs needed to generate and verify SigStore root keys and create signed TUF metadata. 
+This directory contains the programs needed to generate and verify Sigstore root keys and create signed TUF metadata. 
+
+## TUF Repository Structure
+
+The current published repository metadata lives in the [repository](/repository/repository) subfolder of this GitHub repository. In this repository, you will find the top-level TUF metadata files, delegations, and target files. 
+
+* [root.json](repository/repository/root.json): This is the current `root.json`. It is signed by at least 3 out of the 5 [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders). Other signing keys endorsed by this root include:
+  * A set of 5 targets keyholders (the root keyholders).
+  * An online [snapshotting key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L87-L97) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot`. You can verify the hex-encoded public key value with the following code:
+  ```
+  $ gcloud kms keys versions get-public-key 1 --key snapshot --keyring sigstore-root --location global --project project-rekor | openssl ec -pubin -noout -text 
+  ```
+  * An online [timestamping key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L32-L42) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp`
+   ```
+  $ gcloud kms keys versions get-public-key 1 --key timestamp --keyring sigstore-root --location global --project project-rekor | openssl ec -pubin -noout -text 
+  ```
+* [targets.json](repository/repository/targets.json): This is the list of trusted `targets.json` endorsed by the 5 root keyholders. It includes:
+  * [fulcio_v1.crt.pem](repository/repository/targets/artifact.pub): This is the [Fulcio](https://github.com/sigstore/fulcio) root certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running root CA to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert`
+  * [fulcio.crt.pem](repository/repository/targets/artifact.pub): This is the 
```

Comment:
Finish sentence "This is the Fulcio root certificate used with an older instance of Fulcio. Can verify old certificates, but no longer used to sign newly issued certificates."

Reply:
Haha thank you for this catch!